ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -87,6 +90,26 @@
       .put("/submit/sslreports/", "1585144") //
       .build();
 
+  private static final String REDACTED_SEARCH_CODE_VALUE = "other";
```

Comment:
There is ongoing discussion about whether we should use a different value here in order to allow distinguishing client-side sanitization (which uses "other") and pipeline-side sanitization.